java/src/jmri/Block.java

```diff
@@ -152,6 +152,20 @@ public String toDebugString() {
         return result;
     }
 
+    @Override
+    public ArrayList<NamedBeanUsageReport> getUsageReport(NamedBean bean) {
```

Comment:
It's better to return a List than an ArrayList, since List is an interface and ArrayList is a concrete class.

Reply:
Changed in next commit.